ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogger.java

```diff
@@ -164,7 +195,9 @@ public static void deleteOldLogs(int numberOflogsToKeep, String directory)
     */
    public boolean logSession(String logDirectory)
    {
-      String sessionDirectory = generateALogFolderName(logDirectory);
+      String timestamp = dateFormat.format(new Date()).split("_")[0];
```

Comment:
maybe make a new dateFormat that's for the directory? something like `SimpleDateFormat directoryDateFormat = new SimpleDateFormat("yyyyMMdd");`

Reply:
Ahhh ok ya good idea, let me try that